tests/benchdnn/inputs/rnn/test_augru_ci

```diff
@@ -4,9 +4,6 @@
 --activation=UNDEF
 --direction=left2right
 --skip-nonlinear=false
---l=1
---t=1,2
---mb=1,2
```

Comment:
Are `test_augru_ci`, `test_gru_ci`, and `test_lstm_ci` even necessary?  I did a similar refactoring of [test_rnn_ci](https://github.com/oneapi-src/oneDNN/blob/main/tests/benchdnn/inputs/rnn/test_rnn_ci) due to test time on on GPU in 11de372d25f1ae7f3a4b0453095028cc41edded5. With the removal of the `-l`, `-t`, and `--mb` parameters, the test sets looks nearly identical to what is already covered by test_rnn_ci.

Reply:
The are likely not very-very needed, just to keep them identical to Nightly counterparts. Let's reconsider the files once we have something generated working.